config/config.go

```diff
@@ -45,3 +47,25 @@ func Exists(path string) bool {
 
 	return info.Mode().IsRegular()
 }
+
+func TryLoadRootConfig(dir string) (found bool, cfg hcl.Config, err error) {
+	path := filepath.Join(dir, Filename)
+	_, err = os.Stat(path)
+	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
Seems this makes me wonder if it would not be better to model this in the same fashion, like you load the root config and one of the possible errs is "not found", then callers can handle this differently if they want, instead of using the boolean.